.github/ISSUE_TEMPLATE/rebase.md

```diff
@@ -52,13 +52,19 @@ Branching is when a new stream is "branched" off of `rawhide`. This eventually b
 
 - [ ] Bump `releasever` in `manifest.yaml`
 - [ ] Update the repos in `manifest.yaml` if needed
-- [ ] Run `cosa fetch --update-lockfile`
+- [ ] Run `cosa fetch --dry-run --update-lockfile`
+    - this updates the x86_64 lockfile - the others will get updated when `bump-lockfile` runs.
+    - in the future we may support [this](https://github.com/coreos/coreos-assembler/issues/3088) in `cosa fetch` directly
 - [ ] PR the result
 
+- [ ] Re-enable `next-devel` if needed ([docs](https://github.com/coreos/fedora-coreos-pipeline/tree/main/next-devel))
+- [ ] Disabled `branched` stream since it is no longer needed.
+    - Update [config.yaml](https://github.com/coreos/fedora-coreos-pipeline/blob/main/config.yaml) to comment out the `branched` stream definition.
+
 ### Ship rebased `next`
 
 - [ ] Ship `next`
-- Set a new update barrier for N-2 on `next`. In the barrier entry set a link to [the docs](https://docs.fedoraproject.org/en-US/fedora-coreos/update-barrier-signing-keys/). See [discussion](https://github.com/coreos/fedora-coreos-tracker/issues/480#issuecomment-631724629).
+- [ ] Set a new update barrier for the final release of N-1 on `next`. In the barrier entry set a link to [the docs](https://docs.fedoraproject.org/en-US/fedora-coreos/update-barrier-signing-keys/). See [discussion](https://github.com/coreos/fedora-coreos-tracker/issues/480#issuecomment-1247314065)
```

Comment:
Hmm, why did we change this from N-2 to N-1?

Reply:
A few months ago I grabbed you and @bgilbert and we had this discussion. We decided that the current instructions weren't clear and what we had been doing had been inconsistent. We decided that rather than retroactively add barriers to old releases that we would just go with the policy of adding a barrier to the last build for `N-1` in the stream.

Reply:
Ahh gotcha, yup. This does ring a bell and I see now it was recorded as a separate follow-up comment in the same tracker issue.